### PR TITLE
handle correctly checkbox fields

### PR DESCRIPTION
**Pull request by smeeckaert**
There is a little bug that doesn't handle well unchecked checkboxes when serializing forms informations resulting in the impossibility to remove the captcha.

### CODE REVIEW

File: static/js/admin/insert_update.js [hunk 1]
```diff
@@ -85,7 +85,9 @@ define(
                     // If the input name does not match the regex, put the value in the post as usual
                     // _csrf is handled here
                     if ($this.attr('name').indexOf(fieldPrefix) != 0) {
-                        ajaxData[$this.attr('name')] = $this.val();
+                        if ($this.attr('type') != 'checkbox' || $this.is(":checked")) {
```
Comment (jguyomard) [inline static/js/admin/insert_update.js]: Using $this.not(':checkbox') seems better than " $this.attr('type') != 'checkbox' "

Comment (smeeckaert) [inline static/js/admin/insert_update.js]: :checkbox is not recommanded so i used jQuery recommandations instead.

http://api.jquery.com/checkbox-selector/

**Comment by jguyomard** [comment]
:+1:

**Comment by Foine** [comment]
:+1:

**Pull request merged**